src/settings/mailaddress/MailAddressTable.ts

```diff
@@ -9,24 +9,25 @@ import { Icons } from "../../gui/base/icons/Icons.js"
 import { showProgressDialog } from "../../gui/dialogs/ProgressDialog.js"
 import { ExpanderButton, ExpanderPanel } from "../../gui/base/Expander.js"
 import { attachDropdown, DropdownButtonAttrs } from "../../gui/base/Dropdown.js"
-import { showNotAvailableForFreeDialog } from "../../misc/SubscriptionDialogs.js"
+import { showNotAvailableForFreeDialog, showPlanUpgradeRequiredDialog } from "../../misc/SubscriptionDialogs.js"
 import { assertMainOrNode } from "../../api/common/Env.js"
 import { IconButtonAttrs } from "../../gui/base/IconButton.js"
 import { ButtonSize } from "../../gui/base/ButtonSize.js"
 import { AddressInfo, AddressStatus, MailAddressTableModel } from "./MailAddressTableModel.js"
 import { showAddAliasDialog } from "./AddAliasDialog.js"
 import { locator } from "../../api/main/MainLocator.js"
+import { UpgradeRequiredError } from "../../api/main/UpgradeRequiredError.js"
+import Stream from "mithril/stream"
 
 assertMainOrNode()
 
 export type MailAddressTableAttrs = {
 	model: MailAddressTableModel
+	expanded: Stream<boolean>
```

Comment:
I consider streams in attrs antipattern

Reply:
I will fix it

Reply:
... how did that get there then

Reply:
huh? it was @armhub and @nokhub

Reply:
github is showing this before the commits that added this code

Reply:
I'm not saying I don't believe you, I'm saying github is messing up

Reply:
I was just trying to figure out where it comes from